[2023-03-08T09:13:18.718981+00:00] php.DEBUG: Warning: filemtime(): stat failed for /Users/paul/Desktop/sae401test/sae401/config/routes/framework.yaml {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/Users/paul/Library/CloudStorage/OneDrive-Personnel/IUT2/SAE401/sae401/vendor/symfony/config/Resource/FileResource.php","line":58,"trace":[{"file":"/Users/paul/Library/CloudStorage/OneDrive-Personnel/IUT2/SAE401/sae401/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php","line":44,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2023-03-08T09:13:18.810370+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET https://127.0.0.1:8000/"" at /Users/paul/Library/CloudStorage/OneDrive-Personnel/IUT2/SAE401/sae401/vendor/symfony/http-kernel/EventListener/RouterListener.php line 127 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET https://127.0.0.1:8000/\" at /Users/paul/Library/CloudStorage/OneDrive-Personnel/IUT2/SAE401/sae401/vendor/symfony/http-kernel/EventListener/RouterListener.php:127)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /Users/paul/Library/CloudStorage/OneDrive-Personnel/IUT2/SAE401/sae401/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:185)"} []
[2023-03-08T09:13:18.928552+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET https://127.0.0.1:8000/"" at /Users/paul/Library/CloudStorage/OneDrive-Personnel/IUT2/SAE401/sae401/vendor/symfony/http-kernel/EventListener/RouterListener.php line 127 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET https://127.0.0.1:8000/\" at /Users/paul/Library/CloudStorage/OneDrive-Personnel/IUT2/SAE401/sae401/vendor/symfony/http-kernel/EventListener/RouterListener.php:127)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /Users/paul/Library/CloudStorage/OneDrive-Personnel/IUT2/SAE401/sae401/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:185)"} []
